### ISSUE TITLE
Set non-zero exit status if there are errors

### ISSUE THREAD

**Issue by Gankra** [Gankra/abi-checker#10]
Right now I don't do this because we don't have #9 and have checked in tests that fail for i128.

**Comment by bjorn3** [comment]
Rust's compiletest has a `// known-bug` directive since recently. This causes the test results to be inverted, so if a test failed, it would pass as the failure is already known, but if it passes, the test now fails so that you can mark it fixed. I think it would make sense to do something similar for the i128 tests.

**Comment by Gankra** [comment]
yep, #9 discusses the way I would do that with a slant more towards tests being shared between projects

**Comment by Gankra** [comment]
done by #13
